apis/beacon/blocks/sync_committee_rewards.yaml

```diff
@@ -0,0 +1,59 @@
+get:
+  operationId: getSyncCommitteeRewards
+  summary: Get sync committee rewards
+  description: Retrieves rewards info for sync committee members.
+  tags:
+    - Beacon
```

Comment:
I’ve added a “Rewards” tag for block and attestation apis. Might want to add it here too